components/automate-ui/src/app/components/tabs/tabs.component.scss

```diff
@@ -53,3 +53,33 @@
     background: $chef-light-grey;
   }
 }
+
+.beta-tag {
+  width: 24px;
+  height: 12.8px;
+  left: 937px;
+  top: 221.6px;
+  background: #000000;
+  border-radius: 8px;
+  padding-left: 3px;
+
+  span {
+    width: 200px;
+    height: 16px;
+    left: 939px;
+    top: 218px;
+
+    font-family: Mulish;
```

Comment:
I see lots of uses of `Muli` but no use of `Mulish` in the code base--why is a new font needed here? Or is it a typo...?

Reply:
No, it's not a typo but as per the new UX design we will use this font-family but for now, I have removed this font-family as we need to keep the font-family same as we were using earlier.